crowbar_framework/Guardfile

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/ruby
+#
+# More info at https://github.com/guard/guard#readme
+
+guard_opts = {
+  cmd: "bundle exec rspec --fail-fast",
+  all_on_start:   true,
+  all_after_pass: true,
+  failed_mode: :focus
+}
+
+DEBUG = false
+
+def reload(target)
+  puts "-> #{target}" if DEBUG
+  target
+end
+
+def all_specs;      reload "all_specs";      "spec"           end
+def library_specs;  reload "library_specs";  "spec/libraries" end
+def provider_specs; reload "provider_specs"; "spec/providers" end
+
+group :rspec do
+  guard :rspec, guard_opts do
+    watch('app/controllers/application_controller.rb') {
+      "spec/controllers"
+    }
```

Comment:
Because if the `application_controller` changes, all controllers need to be retested.